src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultJsonSerializerFactory.cs

```diff
@@ -1,42 +1,73 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
 using System.Buffers;
 using System.Diagnostics.CodeAnalysis;
 using System.Text.Json;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.Extensions.Caching.Hybrid.Internal;
 
 internal sealed class DefaultJsonSerializerFactory : IHybridCacheSerializerFactory
 {
+    internal static JsonSerializerOptions FieldEnabledJsonOptions { get; } = new() { IncludeFields = true };
+
+    internal JsonSerializerOptions Options { get; }
+
+    public DefaultJsonSerializerFactory([FromKeyedServices(typeof(HybridCache))] JsonSerializerOptions? options = null)
+    {
+#pragma warning disable IDE0079 // unnecessary suppression: TFM-dependent
+#pragma warning disable IL2026, IL3050 // AOT bits
+        Options = options ?? JsonSerializerOptions.Default;
+#pragma warning restore IL2026, IL3050
+#pragma warning restore IDE0079
+    }
+
     public bool TryCreateSerializer<T>([NotNullWhen(true)] out IHybridCacheSerializer<T>? serializer)
     {
-        // no restriction
-        serializer = new DefaultJsonSerializer<T>();
+        // no restriction - accept any type
+
+        var options = Options;
```

Comment:
```suggestion
        JsonSerializerOptions options = Options;
```

Reply:
resolved in https://github.com/dotnet/extensions/pull/5979/commits/b7dd36294b1f68a2e6a3f4e0cd26094cfa507d18, thanks